### PR TITLE
Last PR of the sprint before dev to master

### PR DESCRIPTION
**Pull request by Antropom**
Fixed cards numbers (they got out of the card because of the flexboxes bad config).

### CODE REVIEW

**Review by jeromeHamilka — DISMISSED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (Froyj) [inline .vscode/launch.json:1]: direction .gitignore ça

**Review by Froyj — APPROVED** [review]
utACK! corriger remarque

**Review by Antropom — COMMENTED** [review]
Review done

File: .vscode/launch.json (hunk 1 above)
Comment (Antropom) [reply .vscode/launch.json:1]: En effet.

**Comment by Antropom** [comment]
Commentaire vu

**Review by jeromeHamilka — APPROVED** [review]

**Pull request merged**